extensions/binding/org.eclipse.smarthome.binding.mqttbroker/README.md

```diff
@@ -0,0 +1,82 @@
+# MqttBroker Binding
+
+    MQTT is a machine-to-machine (M2M)/"Internet of Things" connectivity protocol. It was designed as an extremely lightweight publish/subscribe messaging transport.
+
+This binding allows to list textual configured Mqtt brokers for connection status check purposes as well as configure new broker connections. The hereby setup broker connections are shared with the Eclipse Smarthome framework and can be enumerated and consumed by other bindings.
```

Comment:
Only state it here, but I assume this should be changed in the whole documentation(s):
Mqtt => MQTT